koans/ArrayKoans.php

```diff
@@ -0,0 +1,100 @@
+<?php
+namespace PhpKoans;
+
+use PHPUnit\Framework\TestCase;
+
+defined('__') or define('__', null);
+
+class ArrayKoans extends TestCase
+{
+    /**
+     * @testdox Index 0 is the first element of an array
```

Comment:
In these tests I would delete the comments because the use of comments in a test means that something is wrong with the test.

Reply:
Yes it would be cool! I think that in some other koan already implemented they also give too much information in the comments, I will give the other koans a spin too :)